composer.json

```diff
@@ -23,7 +23,7 @@
     "sort-packages": true
   },
   "require": {
-    "php": "^5.5 || ^7.0",
+    "php": "^5.5|^7.0|^8.0",
```

Comment:
I believe we can simplify this to `>=5.5` at this point (similar to `goog/gax`.)